src/main/java/seedu/algobase/logic/commands/AddCommand.java

```diff
@@ -37,7 +37,7 @@
             + PREFIX_SOURCE + "Kattis";
 
     public static final String MESSAGE_SUCCESS = "New Problem added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This Problem already exists in the algobase";
+    public static final String MESSAGE_DUPLICATE_PROBLEM = "This Problem already exists in the algobase";
```

Comment:
Is your fork too far behind upstream/master? Try to rebase from the upstream before starting working next time.